pages/resources/quests.mdx

```diff
@@ -0,0 +1,518 @@
+{
+  "name": "Cybernetic Headgear",
+  "name_with_article": "a Cybernetic Headgear Decoration"
+}
+```
+
+## Endpoints
+
+<RouteHeader method="GET" url="/quests/@me">
+  Get Available Quests
+</RouteHeader>
+
+Returns a list of all available [quests](#quest-object)
+
+##### Example Response
+
+```json
+{
+  "quests": [
+    {
+      "id": "alien-quest",
+      "config": {
+        "quest_id": "alien-quest",
+        "expires_at": "2077-01-01T11:59:59+00:00",
+        "stream_duration_requirement_minutes": 30,
+        "messages": {
+          "quest_name": "👽",
+          "reward_name": "Alien Cybernetics",
+          "reward_name_with_article": "the Alien Cybernetics",
+          "reward_redemption_instructions_by_platform": {
+            "0": "To redeem, kindly beg Dolfies to give them to you 🥺"
+          },
+          "game_title": "Cyberalien 2077",
+          "game_publisher": "Dolfies Studios"
+        },
+        "application_id": "yep",
+        "application_name": "Cyberalien 2077",
+        "get_game_link": "https://docs.discord.sex",
+        "reward_code_platforms": [0],
+        "reward_code_expires_at": "2077-01-01T11:59:59+00:00",
+        "colors": {
+          "primary": "#E944D4",
+          "secondary": "#5318A7"
+        },
+        "assets": {
+          "reward_tile": "reward.png",
+          "hero": "hero.png",
+          "quest_bar_hero": "quests_bar_hero.png",
+          "game_tile": "game_tile.png",
+          "logotype": "game_logotype.png"
+        }
+      },
+      "user_status": {
+        "user_id": "222069018507345921",
+        "quest_id": "alien-quest",
+        "enrolled_at": "2077-01-01T11:59:59+00:00",
+        "completed_at": "2077-01-01T11:59:59+00:00",
+        "claimed_at": "2077-01-01T11:59:59+00:00",
+        "claimed_tier": null,
+        "last_stream_heartbeat_at": "2077-01-01T11:59:59+00:00",
+        "stream_progress_seconds": 1800,
+        "dismissed_quest_content": 0,
+        "progress": {}
+      }
+    }
+  ]
+}
+```
+
+<RouteHeader method="POST" url="/quests/{quest.id}/enroll">
+  Enroll Into A Quest
+</RouteHeader>
+
+Enrolls into/accepts a [quest](#quest-object) and returns a [quest user status object.](#quest-user-status-object)
+
+##### Example Response
+
+```json
+{
+  "user_id": "222069018507345921",
+  "quest_id": "alien-quest",
+  "enrolled_at": "2077-01-01T11:59:59+00:00",
+  "completed_at": null,
+  "claimed_at": null,
+  "claimed_tier": null,
+  "last_stream_heartbeat_at": null,
+  "stream_progress_seconds": 0,
+  "dismissed_quest_content": 0,
+  "progress": {}
+}
+```
+
+##### Example Response
+
+<RouteHeader method="POST" url="/quests/{quest.id}/claim-reward">
+  Claim A Quest's Reward
+</RouteHeader>
+
+Claims the quest's reward and sets both the `completed_at` and `claimed_at` fields of the [User Status Object](#quest-user-status-object) to the current timestamp.
+
+Note that this endpoint _requires_ a valid captcha key.
+
+##### Example Response (Avatar Decorations)
```

Comment:
Response structure should be documented here (tho entitlements and other store stuff are currently undocumented, you don't have to document those)